private/rules/java_export.bzl

```diff
@@ -61,15 +61,14 @@ def java_export(
         and so may contain any valid parameter for that rule.
     """
 
-    tags = tags + ["maven_coordinates=%s" % maven_coordinates]
```

Comment:
Previously, both the `java_library` and the `maven_project_jar` had the maven coordinates added in their tag. With this change, only the former includes it. Please add the coordinates tag back to `maven_project_jar` too

Reply:
Will do

Reply:
Should be fixed now